scripts/ci/ci_prepare_backport_packages.sh

```diff
@@ -20,14 +20,47 @@ export PYTHON_VERSION=${PYTHON_VERSION:-3.6}
 # shellcheck source=scripts/ci/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
+LIST_OF_DIRS_FILE=$(mktemp)
+
+cd "${MY_DIR}/../../airflow/providers" || exit 1
+
+find . -type d | sed 's/.\///' | sed 's/\//\./' | grep -E 'hooks|operators|sensors|secrets' \
```

Comment:
This command gives inconsistent results regarding the dot and slash. Is this expected?
``` 
redis.sensors
redis.operators
redis.hooks
google.ads/operators
google.ads/hooks
google.marketing_platform/sensors
google.marketing_platform/operators
google.marketing_platform/hooks
```

Reply:
Fixed. Was not intentional but it worked anyway ;)